src/ui/application-window.vala

```diff
@@ -395,10 +451,26 @@ namespace Peek.Ui {
 
     private void update_input_shape () {
       // Set an input shape so that the recording view is not clickable
+
+
       var window_region = create_region_from_widget (recording_view.get_toplevel ());
       var recording_view_region = create_region_from_widget (recording_view);
       window_region.subtract (recording_view_region);
 
+      //popover menu
+      if (pop_format.visible) {
+	    var theme = new GLib.Settings ("org.gnome.desktop.interface");//grab the users' settings
+	    var theme_name = theme.get_string ("gtk-theme");//find the users' theme
+	    var pop_style=pop_format.get_style_context();
+	    if ( theme_name == "Ambiance" ){
+		    pop_style.add_class(Gtk.STYLE_CLASS_TITLEBAR);
```

Comment:
What happens if we add this for all themes? Does it do any harm?

Reply:
It does not always behave correctly I have tested with over 10 themes and it is really hit and miss, The only reason I have it set for ambiance because the default for the popover menu was white and it just looked horrible. But now we have an easy way to make changes for specific themes :)